posixlib/src/main/scala/scala/scalanative/posix/locale.scala

```diff
@@ -0,0 +1,17 @@
+package scala.scalanative
+package posix
+
+import scala.scalanative.unsafe._
+
+@extern
+object locale {
+
+  /* This file/object is a less-than-minimal implementation.
+   * It provides only the type local_t. This allows a common
+   * definition of the type to be used by the several files
+   * required by POSIX to define that type.
+   */
```

Comment:
Oops, did you want to do this as a scaladoc?

Reply:
Good question.  I was on the fence.  In the end, my thinking was that the comment
was more of a 'file-level' devo comment than Scaladoc.   The limited nature
of the file is also why I did not add it to the Posixlib ReadTheDocs guide.

I can make it Scaladoc is you advise. I can be influenced to the other side
of the fence.

Reply:
I wasn't really able to understand what it was saying as a devo comment. If it is something that someone using the type needs to know then I would add it to scaladoc.